packages/ListBox/src/store/Provider.js

```diff
@@ -10,26 +9,19 @@ function initializeState(props) {
   const {
     childrenOptions,
     height,
```

Comment:
`height` is also a "controlled" prop that doesn't affect the internal state, but I was warned that `it might be more difficult to refactor since it's used in several calculations, so I didn't touch it.

Reply:
I  was similarly worried about `isMulti` being a can-of-worms...